proof/refine/AARCH64/ArchAcc_R.thy

```diff
@@ -684,11 +686,13 @@ next
     apply (simp add: pt_bits_def)
     apply (prop_tac "level_type (nlevel + 1) = NormalPT_T")
      apply (drule max_pt_level_enum)
-     apply (auto simp: level_defs split: if_split_asm)[1]
+     (* Unfolding here, because the level_enum only produces a contradiction in one of the branches
+        when PA_40 is set. *)
```

Comment:
Not sure about this change, but I think it would be better to say exactly what you're talking about unfolding so it's still obvious when we see this comment in the future without the context of the PR.
```suggestion
     (* Unfolding config_ARM_PA_SIZE_BITS_40_def here, because the level_enum only produces a contradiction in one of the branches
        when PA_40 is set. *)
```